docs/examples/README.md

```diff
@@ -1,4 +1,5 @@
 This folder contains examples of files used as input to proj-frontiers
 
 * `egrades.csv` is an example (with fake data) of a roster for a course at UCSB, in the format provided to instructors on the `eGrades` web app.
-* `chico_canvas.csv` is an example (with fake data) of a roster for a course at Chico State University in the format downloadble from Canvas at Chico State.
\ No newline at end of file
+* `chico_canvas.csv` is an example (with fake data) of a roster for a course at Chico State University in the format downloadble from Canvas at Chico State.
+* `example.csv` is an example of a csv file used to create teams. It is intended to be used after `egrades.csv` has been loaded. It will reject one user: `bob@ucsb.edu`, who does not appear on the roster.
```

Comment:
```suggestion
* `email_csv_example.csv` is an example of a csv file used to create teams. It is intended to be used after `egrades.csv` has been loaded. It will reject one user: `bob@ucsb.edu`, who does not appear on the roster.
```